osi_environment.proto

```diff
@@ -324,4 +336,151 @@ message EnvironmentalConditions
         //
         optional uint32 seconds_since_midnight = 1;
     }
+
+    //
+    // \brief Specification of the cloud states
+    //
+    // \note within this message there is the cloud state level
+    //
+    message Clouds
+    {
+        optional CloudStateLevel cloud_state_level = 1;
+
+        // Definition of the cloud state, i.e. cloud state and sky visualization settings.
+        //
+        // The total degree of coverage indicates how large the part of the sky vault is which is covered
+        // with clouds altogether. It is given in eighths, because this division is easier to estimate
+        // for the observer. 0 eighths means that there are no traces of clouds in the sky,
+        // 4 eighths means that the sky is covered with clouds up to half, 8 eighths means that the sky is
+        // completely covered with clouds and no sky blue can be recognized.
+        //
+        // For cloud determination, the observer will choose a location from which he can overlook the entire sky.
+        // When determining the total cloud cover, all clouds or parts of clouds, regardless of height and genus,
+        // that are above his location will be added. Aircraft contrails are also counted as cloud cover if they remain
+        // in the sky for more than 15 minutes. On the other hand, fog is not counted as cloud cover.
+        //
+        // With a little practice, the observer can easily determine the total amount of cloud cover.
+        // Problems occur only with scattered clouds or many single clouds (cumulus).
+        // Their cloud cover is often overestimated. On the other hand,
+        // the coverage of thin ice clouds (cirrus) is often underestimated.[3]
+        //
+        // \par References:
+        // [1] ISO 15469:2004(E) / CIE S 011/E:2003 : Spatial distribution of skylight
+        // [2] https://de.wikipedia.org/wiki/Bewölkung
+        // [3] https://www.dwd.de/DE/service/lexikon/Functions/glossar.html?lv2=100932&lv3=101016
+        //
+        enum CloudStateLevel
```

Comment:
According to the failing build test, all enum types in OSI have to be named starting with the enum name (=CloudStateLevel)
So they all should be named like this: CLOUD_STATE_LEVEL_UNKOWN, CLOUD_STATE_LEVEL_OTHER, etc.

Reply:
Added the name to all ENUMs